db/db.go

```diff
@@ -88,55 +92,63 @@ func (db *DB) Query(query string) (RowResults, error) {
 		}
 		results = append(results, r)
 	}
-	log.Debug(func() string { return "Executed query successfully: " + query })
+	log.Debugf("Executed query successfully: %s", query)
 	return results, nil
 }
 
 // Execute executes the given sqlite statement, of a type that doesn't return rows.
 func (db *DB) Execute(stmt string) error {
+	var s string
+
 	_, err := db.dbConn.Exec(stmt)
-	log.Debug(func() string {
```

Comment:
Yeah, I see the point. I tried this, but somehow this internal log implementation failed to resolve the func() string when used as a parameter and I was getting a weird value (the func description). I can investigate deeper, there must be a walkaround... Please feel free to let me double check before merging, or go ahead and fix it afterwards.